### ISSUE TITLE
Wish:  check on key present

### ISSUE THREAD

**Issue by stappersg** [gura-conf/gura-rs-parser#8]
To avoid runtime errors with output like
```text
thread 'main' panicked at 'IndexMap: key not found', src/main.rs:25:23
```
is some check on key present needed.

In case such functionality already exists, please where at https://docs.rs/gura/0.3.4/gura/ I should find it.

**Comment by stappersg** [comment]
Ah, nice.

I shall check what https://github.com/gura-conf/gura-rs-parser/commit/1e62e660d28ba6e0107f6eed7d6ccb805853cec4 did bring.

**Comment by Genarito** [comment]
Hi! Sounds fair!

I've published the [0.4.0 version](https://docs.rs/gura/0.4.0/gura/) that includes:

1. A new method called [contains_key](https://docs.rs/gura/0.4.0/gura/parser/enum.GuraType.html#method.contains_key) to check if a specific key exists in an object.
2. Now if you index a Gura Object with an invalid key it'll panic with a more informative error saying that's not an object and that you should check with `contains_key` before indexing.

You can check an example of how to use that method in the [basic example](https://github.com/gura-conf/gura-rs-parser/blob/main/examples/basic.rs).

Sorry for the delay, I was waiting for docs.rs to finish building the documentation. Feel free to reopen the issue if you consider that it's not fixed with the new version :)
